index.js

```diff
@@ -39,69 +38,77 @@ function f(d) {
-  
+  if (loading) {
+    $("#timesTouchedText").hide();
+    $("#totalCount").hide();
+    $("#title").hide();
+    $("#footer").hide();
+  }
   let touchCounter = 0;
+  let totalTouches = 0;
+  let faceAlreadyTouched = false;
+  let faceCurrentlyTouched = false;
+
   const mobile = isMobile();
   const detectorParams = {
-    renderPointCloud: !mobile,
     width: mobile ? undefined : VIDEO_WIDTH,
     height: mobile ? undefined : VIDEO_HEIGHT,
-    timeout: 500,
-    renderPointCloud: true,
+    renderPointCloud: false,
+    timeout: 300,
     renderCanvas: true,
-    renderFaceMesh: true,
     onDetected: () => { touchCounter++; },
-    onRender: () => { stats.begin(); },
+    onRender: () => { 
+      if (loading){
+        $("#timesTouchedText").show();
+        $("#totalCount").show();
+        $("#title").show();
+        $("#footer").show();
+        let el = document.getElementById('loading-animation');
+        el.parentNode.removeChild(el);
+
+        const gui = new dat.GUI();
+        gui.add(state, 'frame timeout', 100, 1000).onChange((value) => { detector.update({ timeout: value }); });
+        loading = false;
+      }
+    },
     onRendered: (result) => {
       const d = result.minDistance;
       const handBox = result.handBox;
       const faceBox = result.faceBox;
       const deltaVolume = result.deltaVolume;
-
-      document.querySelector('#distance').innerText = d
-        ? `Closest ||p||: ${f(d.d)}, Δx: ${f(d.x)}, Δy: ${f(d.y)}, Δz: ${f(d.z)}`
-        : `Closest ||p||: Undefined`;
-
-      document.querySelector('#intersection').innerText =
-        `Volume intersected: ${deltaVolume}`;
-      document.querySelector('#deltaCenter').innerText = d
-        ? `Center bounding box ||p||: ${f(Math.sqrt(Math.pow(handBox.xCenter - faceBox.xCenter, 2) + Math.pow(handBox.yCenter - faceBox.yCenter, 2) + Math.pow(handBox.zCenter - faceBox.zCenter, 2)))} Δx:${f(handBox.xCenter - faceBox.xCenter)} Δy:${f(handBox.yCenter - faceBox.yCenter)} Δz:${f(handBox.zCenter - faceBox.zCenter)}`
-        : `Center bounding box: Undefined`;
-
-      document.querySelector('#detection').innerText =
-        `Detection: ${result.detected ? 'Yes' : 'No'}`;
-      stats.end();
+  
+      faceCurrentlyTouched = result.detected;
     }
   };
 
-  // Check every 5 secs with at least three touches
+  // Check every second with at least three touches
   workerTimers.setInterval(() => {
-    if (touchCounter > 2 && Notification.permission === 'granted') {
-      new Notification('You touched your face!');
+    if (touchCounter >= 2 && Notification.permission === 'granted' && !faceAlreadyTouched) {
+      new Notification('🤭 You touched your face! 🤭');
+      $("#face-touch-alert").show();
+      totalTouches++;
+      document.querySelector('#totalCount').innerText = totalTouches;
+      document.querySelector('#timesTouchedText').innerText = totalTouches === 1 ? 'time touched' : 'times touched';
+      window.document.title = '😱'
```

Comment:
Forgot to say the other day. It's probably better to have a name in the title instead of emoji?  Maybe change the favicon when people touch their face, or have a number badge on the icon when people touch their face? I am saying this because if someone were to bookmark this page, they will have the emoji as the name which doesn't seem ideal.